docs/src/pages/_document.page.tsx

```diff
@@ -26,7 +26,7 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
       font-src 'self' data:;
       frame-src 'self' *.codesandbox.io;
       img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net;
-      connect-src 'self' *.shortbread.aws.dev amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net ;
+      connect-src 'self' *.shortbread.aws.dev amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net https://*.algolia.net https://*.algolianet.com;
```

Comment:
Do we need 'https://' here? I don't see it in other domains in CSP

Reply:
The others probably should have the https, it's more secure to be specific with the protocol.

Reply:
Ok good to know!